models/sipnet/R/split.inputs.SIPNET.R

```diff
@@ -0,0 +1,46 @@
+## split clim file into smaller time units to use in KF
+##' @title split.inputs.SIPNET
+##' @name  split.inputs.SIPNET
+##' @author Mike Dietze and Ann Raiho
+##' 
+##' @param multi.settings
+##' @param start.time
+##' @param stop.time
+##' @description Splits climate met for SIPNET
+##' 
+##' @return file split up climate file
+##' @export
+##' 
+split.inputs.SIPNET <- function(settings,start.time,stop.time){
+  
+  
+  start.day<-strftime(start.time, format = "%j")
+  start.year<-strftime(start.time,"%Y")
+  
+  end.day<-strftime(stop.time, format = "%j")
+  end.year<-strftime(stop.time,"%Y")
+  
+  full.met <- c(settings$run$inputs$met$path)
+  new.met  <- file.path(settings$rundir,basename(full.met))
+  file.copy(full.met,new.met)
```

Comment:
Not sure why you're copying the met before splitting it rather than just opening the original file and writing out the subset.